src/components/app-configure/access/CreateDelegate.vue

```diff
@@ -0,0 +1,161 @@
+<script lang="ts" setup>
+import { ref } from 'vue'
+
+import VButton from '@/components/lib/VButton/VButton.vue'
+import VCard from '@/components/lib/VCard/VCard.vue'
+import VDropdown from '@/components/lib/VDropdown/VDropdown.vue'
+import VOverlay from '@/components/lib/VOverlay/VOverlay.vue'
+import VSeperator from '@/components/lib/VSeperator/VSeperator.vue'
+import VStack from '@/components/lib/VStack/VStack.vue'
+import VTextField from '@/components/lib/VTextField/VTextField.vue'
+import { useToast } from '@/components/lib/VToast'
+import { createDelegate } from '@/services/gateway.service'
+import { useAppId } from '@/use/getAppId'
+
+const emit = defineEmits(['close', 'generateKey', 'created'])
+
+const props = defineProps({
+  delegateKeys: {
+    type: Array,
+    required: true,
+  },
+})
+
+const loader = ref({
+  show: false,
+  message: '',
+})
+
+const toast = useToast()
+const appId = useAppId()
+const delegateName = ref('')
+const selectedDelegateAddress = ref('')
+
+function showLoader(message: string) {
+  loader.value.show = true
+  loader.value.message = message
+}
+
+function hideLoader() {
+  loader.value.show = false
+  loader.value.message = ''
+}
+
+async function onSubmit() {
+  if (!delegateName.value.length || !selectedDelegateAddress.value.length) {
```

Comment:
Prefer avoiding implicit coercion is conditionals.

```suggestion
  if (delegateName.value === "" || selectedDelegateAddress.value === "") {
```

Reply:
https://github.com/arcana-network/developer-dashboard/pull/156/commits/2981de3c4a5362ec0e344c13d12de4efb01075b4

Reply:
Should we also trim it?